src/tests/bitsets/test_stdlib_bitset_64.f90

```diff
@@ -0,0 +1,744 @@
+        end if
+
+        log7 = set5
+        if ( size(log7) /= 64 ) then
+            error stop procedure // &
+                ' initialization of logical(int32) with assignment failed' // &
+                ' to set the right size.'
+        else if ( .not. all(log7) ) then
+            error stop procedure // &
+                ' initialization of logical(int32) with assignment failed' // &
+                ' to set the right values.'
+        else
+            write(*,*) 'Initialization of logical(int32) succeeded.'
+        end if
+
+        log8 = set5
+        if ( size(log8) /= 64 ) then
+            error stop procedure // &
+                ' initialization of logical(int64) with assignment failed' // &
+                ' to set the right size.'
+        else if ( .not. all(log8) ) then
+            error stop procedure // &
+                ' initialization of logical(int64) with assignment failed' // &
+                ' to set the right values.'
+        else
+            write(*,*) 'Initialization of logical(int64) succeeded.'
+        end if
+
+    end subroutine test_initialization
+
+    subroutine test_bitset_inquiry()
+        character(*), parameter:: procedure = 'TEST_BITSET_INQUIRY'
+        integer(bits_kind) :: i
+
+        write(*,*)
+        write(*,*) 'Test bitset inquiry: all, any, bits, none, test, and value'
+
+        if ( set0 % none() ) then
+            if ( .not. set0 % any() ) then
+                write(*,*) 'As expected set0 has no bits set'
+            else
+                error stop procedure // ' set0 had some bits set which ' // &
+                    'was unexpected.'
+            end if
+        else
+            error stop procedure // ' set0 did not have none set which ' // &
+                'was unexpected'
+        end if
+
+        call set0 % not()
+        if ( set0 % all() ) then
+            if ( set0 % any() ) then
+                write(*,*) 'As expected set0 now has all bits set'
+            else
+                error stop procedure // ' set0 had no bits set which ' // &
+                    'was unexpected.'
+            end if
+        else
+            error stop procedure // ' set0 did not have all bits set ' // &
+                'which was unexpected'
+        end if
+
+        if ( set1 % any() ) then
+            if ( set1 % all() ) then
+                write(*,*) 'As expected set1 has all bits set'
+            else
+                error stop procedure // ' set1 did not have all bits set ' // &
+                    'which was unexpected.'
+            end if
+        else
+            error stop procedure // ' set1 had no bits set ' // &
+                'which was unexpected'
+        end if
+
+        call set0 % not()
+        do i=0, set0 % bits() - 1
+            if ( set0 % test(i) ) go to 100
+        end do
+
+        write(*,*) 'As expected set0 had no bits set.'
+
+        go to 110
+
+100     error stop procedure // ' against expectations set0 has ' // &
```

Comment:
`100` is only used once Therefore, I suggest to replace the `go to 100` by the `error stop` directly in te code.